### PR TITLE
Adds the start of requirements page styling

### PR DESCRIPTION
**Pull request by theresaanna**
Implements:
screen shot 2017-03-22 at 2 56 38 pm

Addresses but does not complete https://github.com/18F/omb-eregs/issues/90.

This is not exactly to mocks yet, but brings us a step further in implementing the design.

### CODE REVIEW

File: ui/assets/css/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+// https://github.com/18F/web-design-standards/blob/develop/src/stylesheets/core/_variables.scss
+
+$color-primary:              #0071bc !default;
+$color-primary-darker:       #205493 !default;
+$color-primary-darkest:      #112e51 !default;
+
+$color-primary-alt:          #02bfe7 !default;
+$color-primary-alt-dark:     #00a6d2 !default;
+$color-primary-alt-darkest:  #046b99 !default;
+$color-primary-alt-light:    #9bdaf1 !default; // lighten($color-primary-alt, 60%)
```
Comment (donjo) [inline ui/assets/css/_colors.scss]: FYI the comments that describe how the colors were generated here aren't actually true anymore. I've been meaning to submit a PR to the WDS to get the removed but keep forgetting 

screen shot 2017-03-22 at 2 45 59 pm

File: ui/assets/css/_fonts.scss [hunk 2]
```diff
@@ -50,3 +50,9 @@
    font-weight: 400;
    src: url('/static/font/Merriweather-Italic.ttf');
  }
+
+ @font-face {
```
Comment (donjo) [inline ui/assets/css/_fonts.scss]: nitpicking here but can we keep these in order of font-weight to make it easier to scan through if something needs changed?

File: ui/components/requirements.jsx [hunk 3]
```diff
@@ -8,17 +8,34 @@ import Pagers from './pagers';
 import FilterList, { fetchKeywords, fetchPolicies } from './filter-list';
 
 function Requirement({ requirement }) {
-  return <li className="req">{requirement.req_id}: {requirement.req_text}</li>;
+  return (
```
Comment (donjo) [inline ui/components/requirements.jsx]: 🙌

File: ui/assets/css/_requirements.scss [hunk 4]
```diff
@@ -1,7 +1,22 @@
...
+
+  .req-id {
+    color: $color-gray;
+    font-family: "Merriweather", serif;
+  }
+
+  .req-text {
+    color: $color-gray-dark;
+    font-family: "Merriweather", serif;
+    font-weight: 300;
```
Comment (cmc333333) [inline ui/assets/css/_requirements.scss]: Not for this round, but does it make sense to put these font family, weight, size, etc. in a mixin? I assume we'll want to reuse for consistency.

File: ui/assets/css/_fonts.scss (hunk 2 above)
Comment (theresaanna) [inline ui/assets/css/_fonts.scss]: That's not nitpicking, that makes a lot of sense! Will do.

File: ui/assets/css/_requirements.scss (hunk 4 above)
Comment (theresaanna) [inline ui/assets/css/_requirements.scss]: Yes, for sure. I'll actually do that before merging.

**Pull request merged**